content/docs/command-reference/plots/diff.md

```diff
@@ -47,7 +47,21 @@ please see `dvc plots`.
 
 - `--targets <path>` - specific metric files to visualize. These must be listed
   in a [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) file
-  (see the `--plots` option of `dvc run`).
+  (see the `--plots` option of `dvc run`). When you are specifying multiple
+  arguments for `--targets` and `revisions`, use `--` after all the arguments
+  for `--targets`:
+
+  ```dvc
+  $ dvc plots diff --targets t1.json t2.csv -- HEAD v1 v2
+  ```
+
+  Without `--`. this statement will show an error as argument parser confuses
+  `revisions` as arguments for `--targets` option. Alternatively, you can also
+  run above statement as:
```

Comment:
Just
```suggestion
Alternatively, you can also run above statement as:
```

Reply:
I wonder is this should be the first recommendation though, not `--`. I also wonder which other commands can benefit from the `--` syntax.

Reply:
WDYT @imhardikj ?

Reply:
I think order is correct. As mentioned in https://github.com/iterative/dvc.org/pull/1544#discussion_r451395365 it's almost natural instinct to give `targets` first and then `revisions`.

Reply:
`--` will be used only when a option accepts multiple arguments and at least one positional argument. This can be used in `metrics diff`. But right now `metrics diff` doesn't have any example which shows usage of `--targets` option.

Reply:
But the same note applies to metrics diff? If so please copy it to that reference as well in your new PR, or a subsequent one 🙂 Adding an targets example (similar to the one here) would be a nice addition too but not necessary.